.github/workflows/regression.yml

```diff
@@ -7,6 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         pg:
+          - 16
```

Comment:
The file `doc/pg_repack.rst` must be updated too to declare the next version pg16-compatible.